pkg/interactor/data.go

```diff
@@ -0,0 +1,110 @@
+package interactor
+
+import (
+	"log"
+	"strconv"
+	"time"
+
+	. "github.com/Igoraamc/knot-cloud-storage/pkg/entities"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type DataDAO struct {
+	Server   string
+	Database string
+}
+
+var db *mgo.Database
+
+const (
+	COLLECTION = "things"
+)
```

Comment:
This looks good, just got a few formatting comments :stuck_out_tongue: 

1. [Function Grouping and Ordering](https://docs.google.com/document/d/1w61-0sp_QkuaCI3qepc9yxh0AEkCBuS0bOyKSahsOkQ/edit#heading=h.fwxbbk14050i)
2. [MixedCaps or mixedCaps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps)
3. [Single/Multiple declarations](https://blog.learngoprogramming.com/learn-golang-typed-untyped-constants-70b4df443b61)

Let us know if you got any suggestions about the points above :smile: 

```suggestion
const collection = "things" //capitalize if it's exported: 'Collection' (update all occurrences)

var db *mgo.Database

type DataDAO struct {
	Server   string
	Database string
}
```

Reply:
Done!